python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -401,6 +401,9 @@ def resolve_inner(self, asset_graph: AssetGraph) -> AbstractSet[AssetKey]:
     def to_serializable_asset_selection(self, asset_graph: AssetGraph) -> "AssetSelection":
         return self
 
+    def __str__(self) -> str:
+        return "all materializable assets"
```

Comment:
I think we're going to eventually want to turn this into something more machine parseable. This is me being a little bit lazy and deferring what exactly that should be to the future.